pkg/gateway/multiparts/main_test.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/ory/dockertest/v3"
 
+	_ "github.com/treeverse/lakefs/pkg/kv/mem"
```

Comment:
I like the approach where unit tests use 'mem' and system tests use postgres. I think the only reason we used postgres in our unit tests is because we didn't have any other option.
We can take advantage of the new mem implementation and also save a lot of runtime. WDYT?